### DISCUSSION TITLE
Markdown Table Of Contents links no longer work

### DISCUSSION THREAD

**Discussion by amadejpapez** [community/community#21200] [Mobile] [answered]
Markdown Table Of Contents, with links to titles, like:
```
- [a:](#a)
- [b:](#b)
```

no longer works. Clicking on them doesn't move to that title, but seems to move to the top instead.

This worked previously in the iOS app and I don't know which update broke this behavior.

An example can be found in the README here: https://github.com/AlanWalk/markdown-toc#4-configuration

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.73.0 (72950773)
Device: iPhone 8 Plus (iOS 16.0)
</details>

**Comment by Joeridevits** [comment]
Helo i wont thé Learn

**Comment by radar-blip** [comment]
@amadejpapez, You can file a bug report here: https://support.github.com/contact/bug-report

to help get fixed sooner.

**Comment by aconsuegra** [comment]
Thanks for the bug report @amadejpapez. I confirmed that links are not working as intended in our iOS app, I'm going to create an issue in our end so we can track it, and hopefully ship a fix soon.

**Comment by amadejpapez** [comment, marked as answer]
Seems to be fixed with today's update. Thank you :)

**Comment by jacko-yang** [comment]
I am seeing the same issue described in the original post in the Android version of the app (version 1.142.0). I.e., TOC links seem to take you to the top of the page instead of the actual section.
